dashboard/config/libraries/Outbreak.interpreted.js

```diff
@@ -0,0 +1,158 @@
+/* -------------- START OF MONSTER BEHAVIOR DEFINITIONS -------------- */
+// Owned By Curriculum
```

Comment:
should we move these ownership comments into the comment above? e.g. `START OF MONSTER BEHAVIOR DEFINITIONS (Curriculum-owned)`